config/routes.rb

```diff
@@ -1,10 +1,18 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
   # Can be used by load balancers and uptime monitors to verify that the app is live.
   get "up" => "rails/health#show", as: :rails_health_check
 
-  # Defines the root path route ("/")
-  # root "posts#index"
+  root "users#index" # Defines the root path route ("/")
+  get 'users' => 'users#index', as: :users
+  get 'users/:id' => 'users#show', as: :user
+  get 'users/:user_id/posts' => 'posts#index', as: :user_posts
+  get 'users/:user_id/posts/:id' => 'posts#show', as: :user_post
```

Comment:
- [ OPTIONAL ] I think it would be great if you remove this routes since you already added them below so you don't need to redundant the routes so please just keep one way to add the routes since there's no need to repeat them 💪🏻🔥

Reply:
Thank you @Reem-lab for the review and feedback!